src/Eccube/Controller/Admin/Product/ProductController.php

```diff
@@ -736,6 +737,16 @@ public function export(Application $app, Request $request)
             $qb = $app['eccube.service.csv.export']
                 ->getProductQueryBuilder($request);
 
+            // Get stock status
+            $stockStatus = 0;
+            $session = $request->getSession();
+            if ($session->has('eccube.admin.product.search')) {
+                $searchData = $session->get('eccube.admin.product.search', array());
+                if (isset($searchData['stock_status']) && $searchData['stock_status'] === 0) {
```

Comment:
it mean, after show search results  in admin. when click on outofStock , it will be filter again these result. not change search condition?